### PR TITLE
fix(common): Reintroduce missing CHANGELOGs

### PR DESCRIPTION
**Pull request by david-cermak**

### CODE REVIEW

File: components/esp_websocket_client/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Changelog
+
+## [1.0.0](https://github.com/espressif/esp-protocols/commits/996fef7)
+
+### Updated
+
+- esp_websocket_client: Updated version to 1.0.0 Updated tests to run agains release-v5.0 ([996fef7](https://github.com/espressif/esp-protocols/commit/996fef7))
+- esp_websocket_client: * Error handling improved to show status code from server * Added new API `esp_websocket_client_set_headers` * Dispatches 'WEBSOCKET_EVENT_BEFORE_CONNECT' event before tcp connection ([d047ff5](https://github.com/espressif/esp-protocols/commit/d047ff5))
+- unite all tags under common structure py test: update tags under common structure ([c6db3ea](https://github.com/espressif/esp-protocols/commit/c6db3ea))
+- Adds esp_mqtt_cxx component ([1407dfc](https://github.com/espressif/esp-protocols/commit/1407dfc))
```
Comment (gabsuren) [inline components/esp_websocket_client/CHANGELOG.md:10]: Not sure about this

Comment (gabsuren) [inline components/esp_websocket_client/CHANGELOG.md:10]: not sure about this message `Adds esp_mqtt_cxx component` in scope of Websocket changes

Comment (david-cermak) [reply components/esp_websocket_client/CHANGELOG.md:10]: This commit changed internal component reference in `components/esp_websocket_client/examples/CMakeLists.txt`

But I see that it's a bit confusing. I'll delete this line manually.

**Review by gabsuren — APPROVED** [review]

**Comment by gabsuren** [comment]
LGTM

**Comment by david-cermak** [comment]
Thanks for the review @gabsuren

**Pull request merged**